### PR TITLE
Added IceCube for compatibility issues

### LINKED ISSUE #14: Backward compatibility for persistance
The perstiance are not compatible yet with the old gem.

May be we need to add somewhere something like "IceCube = IceCubed". Otherwise we have some trouble with the serialisation store in the database for exemple.

```
recurrence:
  "---\n:start_time:\n  :time: 2021-10-17 22:00:00.000000000 Z\n  :zone: Paris\n:end_time:\n  :time: 2021-10-18 21:00:00.000000000 Z\n  :zone: Paris\n:rrules:\n- :validations: {}\n  :rule_type: IceCube::DailyRule\n  :interval: 1\n:rtimes: []\n:extimes: []\n",
```

### PR DESCRIPTION
**Pull request by Slokilla**
Just an IceCube module that includes the IceCubed module, for projects that still use "IceCube" module.

Close #14

### CODE REVIEW

File: spec/examples/ice_cube_module.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require File.dirname(__FILE__) + '/../spec_helper'
+
+module IceCubed
+    describe 'IceCube' do
+        it "is an alias of IceCubed" do
+        expect(IceCube::VERSION).to be IceCubed::VERgca!SION
+        end
+    end
+end
```
Comment (pacso) [inline spec/examples/ice_cube_module.rb:9]:
```suggestion
end

```

**Comment by Slokilla** [comment]
i can duplicate tests using IceCube module, but i feel like it's a bit too much

**Comment by Frederik-Baetens** [comment]
Does this PR make it so that only the gem name has to be changed from ice_cube to ice_cubed, or will users that make the switch also need to explicitly require the Ice_Cube module created in this PR?

**Comment by Frederik-Baetens** [comment]
> This is good, but we probably need to ensure that some of the functionality also works against the new name rather than just checking the versions match?

I assume most people wouldn't bother renaming ice_cube then, we could just switch all tests back to ice_cube?

**Comment by Slokilla** [comment]
I need more time to check the possibilities of rubygem, i'm an abslute noob concerning ruby environment. When i'll be sure that there is no other solutions, i'll reopen the PR :)

**Comment by pacso** [comment]
Whilst technically we could release a gem with a module named differently to the gem itself ... that is not the agreed practice.

I think this approach was the right way to address it.

**Pull request closed without merging**